### PR TITLE
chore: add dbt-coverage report

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR modifies the documentation workflow to add a check with documentation coverage. Eventually, it can be modified to verify test coverage.

This PR also adds documentation for every dataset.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: models/completion/schema.yml [hunk 1]
```diff
@@ -9,25 +9,60 @@ models:
         description: "Timestamp, to the second, of when this event was emitted"
         data_type: DateTime64(3)
       - name: org
-        data_type: String
+        data_type: String}
```
Comment (bmtcril) [inline models/completion/schema.yml:12]: Looks like a typo here

File: models/grading/schema.yml [hunk 2]
```diff
@@ -61,3 +74,63 @@ models:
...
+        description: "The course key for the course"
+      - name: actor_id
+        data_type: string
+        description: "The xAPI actor identifier"
+      - name: course_grade
+        data_type: float64
+        description: "The most recent grade for the learner"
+
+  - name: fact_learner_course_status
+    description: "One record per learner per course for the most recent approving status"
```
Comment (bmtcril) [inline models/grading/schema.yml:95]: I'm not sure what "approving status" is here? Is this pass/fail?

**Review by bmtcril — COMMENTED** [review]
This is a great start thanks for getting it all in.

File: models/grading/schema.yml (hunk 2 above)
Comment (Ian2012) [reply models/grading/schema.yml:95]: Yes, it's passed or failed

Comment (bmtcril) [reply models/grading/schema.yml:95]: Can you clarify that in the description?

Comment (Ian2012) [reply models/grading/schema.yml:95]: done

**Comment by bmtcril** [comment]
I think maybe `coverage.json` should be added to `.gitignore`? I know for other coverage things it's an artifact file that gets generated as part of the run but isn't kept around between runs.

**Review by bmtcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/aspects-dbt&entry.867055334=https://github.com/openedx/aspects-dbt/pull/51&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-03-04+20:00&entry.1917517419=2024-03-07+15:05&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**